### DISCUSSION TITLE
Cant get Nativewind to work

### DISCUSSION THREAD

**Discussion by ammartiger** [nativewind/nativewind#226] [Q&A] [answered]
I have followed complete instructions but still nativewind does not work. Its an expo project created from scratch

Github repo link for essential files
[https://github.com/ammartiger/noteapp](https://github.com/ammartiger/noteapp
)
**babel.config.js**

`module.exports = function (api) {
  api.cache(true);
  return {
    presets: ["babel-preset-expo"],
    plugins: ["nativewind/babel"],
  };
};
`
**tailwind.config.js**

`// tailwind.config.js
module.exports = {
  content: [
    "./screens/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
    "./App.js",
  ],
  // ...
};
`

**Package.json**

`{
  "name": "notes",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web"
  },
  "dependencies": {
    "expo": "~46.0.9",
    "expo-status-bar": "~1.4.0",
    "nativewind": "^2.0.7",
    "react": "18.0.0",
    "react-native": "0.69.5"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "tailwindcss": "^3.1.8"
  },
  "private": true
}
`

**Comment by marklawlor** [comment, marked as answer]
`class` is not a valid prop due to `class` being a restricted keyword in JavaScript. The correct prop when using React is `className`

**Reply by ammartiger** [reply]
Thanks mark newbie mistakes

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
